samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -42,10 +45,48 @@ private enum SupportedFileType
         Pdf,
     };
 
+    /// <summary>
+    /// Value of `Content` for a `ChatMessage` of type `ChatMessageType.Document`.
+    /// </summary>
+    public class DocumentMessageContent
```

Comment:
Let's continue to put public classes in their own files.

Reply:
fixed